### ISSUE TITLE
createLineage_Knn with umap embedding...?

### ISSUE THREAD

**Issue by Xieeeee** [ChengxiangQiu/tome_code#1]
Hi,
Thanks for sharing the analysis details of this exciting paper! The trajectories construction is very amazing, and I really like the TOME portal you shared for exploration.

One thing I am not sure in `Section2_trajectory_Step1_connection.R` is that it seems you save the umap embedding and then used it for knn to deduce 'progenitor'; To my understanding normally reduction like pca is used for knn. Is the umap here a typo or so?

Best,
Yang

**Comment by ChengxiangQiu** [comment]
Thanks for your comments. In the paper, we used the 3D UMAP features to connect cells across timepoints. To evaluate the robustness of this approach to the choice of co-embedding space, we repeated it using Euclidean distances between cells in PCA space (dims = 30) instead of UMAP space (dims = 3). The resulting edge weights were highly correlated (Pearson correlation coefficient = 0.993). We evaluated the above approach with k parameters (for the k-NN) other than five, and found the resulting edge weights to be highly correlated with those obtained with k = 5 (Pearson correlation coefficients from 0.9994 to 0.9999 for k = 8, 10, 15, 20). If you are running the program in your own project, I would suggest you to try different parameters, to see how the results changing.
